### PR TITLE
Basic IOHelper

### PR DESCRIPTION
**Pull request by tomlazar**

### CODE REVIEW

File: java/io/IOHelper.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+     * @param nFile file to write and read
+     */
+    public IOHelper(String nFile)
+    {
+        inputFile = new File(nFile);
+        outputFile = inputFile;
+    }
+
+    /**
+     * Constructs a new helper with seperate input and output files
```
Comment (Minnow42) [inline java/io/IOHelper.java:29]: Seems like you need to construct a helper for your spelling (sep**a**rate)

File: java/io/IOHelper.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+     * @return Array of N strings from a file where each string is line from the
+     * input file.
+     */
+    public String[] readAllAsArray()
+    {
+      return this.readAll().Split("\n");
+    }
+
+    /**
+     * writes the input to the file, this operation is destructive to the previous contents of the file.
```
Comment (Minnow42) [inline java/io/IOHelper.java:67]: Need a semicolon

File: java/io/IOHelper.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+/**
+ * Created by lazar on 9/8/2016.
+ * Basic helper for simple File IO operations.
+ *
+ * @author Tom Lazar
+ *         (262) 408-0490
+ *         thomas.lazar@marquette.edu
+ */
+public class IOHelper
+{
```
Comment (Minnow42) [inline java/io/IOHelper.java]: CUDDLE

**Review by Minnow42 — CHANGES_REQUESTED** [review]
Fix your braces

File: java/io/IOHelper.java (hunk 2 above)
Comment (tomlazar) [reply java/io/IOHelper.java:67]: what

**Comment by Minnow42** [comment]
Test

**Comment by Minnow42** [comment]
That shit didn’t work

**Review by Minnow42 — APPROVED** [review]
👏🏼

File: java/io/IOHelper.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
+package com.tomlazar.util;
+
+import java.io.*;
```
Comment (TomRaven) [inline java/io/IOHelper.java]: j

**Pull request merged**